src/frontend/qt_sdl/PlatformConfig.cpp

```diff
@@ -110,6 +110,7 @@ ConfigEntry PlatformConfigFile[] =
     {"HKKey_Reset",               0, &HKKeyMapping[HK_Reset],               -1, NULL, 0},
     {"HKKey_FastForward",         0, &HKKeyMapping[HK_FastForward],         -1, NULL, 0},
     {"HKKey_FastForwardToggle",   0, &HKKeyMapping[HK_FastForwardToggle],   -1, NULL, 0},
+    {"HKKey_FullscreenToggle",    0, &HKKeyMapping[HK_FullscreenToggle],    - 1, NULL, 0},
```

Comment:
This is misaligned because of a space between `-` and `1`.

Reply:
done